### DISCUSSION TITLE
🏓 [simulateur salarié] Ajouter les aides à l'embauche

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2613] [Idées]
Le retour des aides à l'embauche. Issue à compléter au fur et à mesure

```[tasklist]
### Aides à l'embauche
- [ ] Ajouter l’aide à l’embauche d’un demandeur d’emploi de + 45 ans en contrat pro
- [ ] Ajouter l'AETH (aide à l'emploi des travailleurs handicapés)

- [ ] https://github.com/betagouv/mon-entreprise/issues/2533
```
Sources
- https://entreprendre.service-public.fr/vosdroits/F15204
- https://travail-emploi.gouv.fr/emploi-et-insertion/mesures-seniors/article/l-aide-a-l-embauche-d-un-demandeur-d-emploi-de-45-ans-et-plus-en-contrat-de

> **Tags** : RQTH ; senior ; jeunes ; plus de 50 ans

Relié : #1895

**Comment by glopezrios** [comment]
Un usager vient de remonter ce besoin, concernant l'absence, dans le simulateur salarié, des aides pour le recrutement de travailleur handicapé, cf. https://entreprendre.service-public.gouv.fr/vosdroits/F15204
